core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -24,11 +24,15 @@ import org.I0Itec.zkclient.exception.ZkNodeExistsException
 import kafka.common.{AdminCommandFailedException, TopicAndPartition}
 import kafka.log.LogConfig
 import LogConfig._
+import kafka.admin.ReassignPartitionsCommand.Throttle
 import org.apache.kafka.common.utils.Utils
 import org.apache.kafka.common.security.JaasUtils
 
 object ReassignPartitionsCommand extends Logging {
 
+  case class Throttle (value: Long, postUpdateDelay: Int = 0)
```

Comment:
A minor nit: could we take a function `postUpdate: () => Unit` and then do the sleep in the test? That seems a bit nicer than having the sleep in the command. Apart from that, this looks OK, I could not think of a better way to detect that the throttling information had propagated.